hail/src/test/scala/is/hail/expr/ir/AggregatorsSuite.scala

```diff
@@ -630,25 +628,6 @@ class AggregatorsSuite extends HailSuite {
       FastIndexedSeq(Call2(0, 1), Call2(1, 1), null))
   }
 
-  @Test def pearsonCorrelationAggregator() {
```

Comment:
Is there a corresponding test in Python?

Reply:
There is a test in python that runs the aggregator over a smaller set (once) and checks against scipy. I can make it run more times against a larger set, perhaps?

Reply:
It looks like there's no null values. Might be good to add a second test with null values.

```
    def test_agg_corr(self):
        ht = hl.utils.range_table(10)
        ht = ht.annotate(x=hl.rand_unif(-10, 10),
                         y=hl.rand_unif(-10, 10))
        c, xs, ys = ht.aggregate((hl.agg.corr(ht.x, ht.y), hl.agg.collect(ht.x), hl.agg.collect(ht.y)))

        scipy_corr, _ = pearsonr(xs, ys)
        self.assertAlmostEqual(c, scipy_corr)
```